hardware/README.md

```diff
@@ -1,6 +1,30 @@
+# Version 2 - improvement proposal
+## Hardware
+### Same components
+* 1 x Raspberry Pi Zero W 
+* 3 x Clicker buttons
+* 1 x Power button
+* 1 x 3000MAh LiPo battery (TBC, see power section below)
+
+### New components
+#### Display
+
+[This kind of display](https://www.waveshare.com/product/raspberry-pi/displays/2.42inch-oled-module.htm) would allow a broader range of info to be available, while retaining a small form factor and lower power consumption. Its ability to be plugged directly on top of the RPi Zero would also improve the wiring (and assembly) a lot.
+
+Waveshare offers quite a range of displays and has solid expertise and documentation, in addition to being fairly competitive in terms of pricing.
+
+### Power supply
 
-# Hardware
-## Components
+As we've established, the Powerboost might be a bit overkill for this specific project.
+
+Option 1 would be to go with a simpler Lipo Charging circuit, as we've discussed together. I'm not sure, however, that a LiPo pack without a booster would be beefy enough to power a RPi Zero -- does Vladimir have any input on this ?
+
+Option 2 would be, IMHO, to power the whole thing with a portable battery pack  (those that are readily available for people with iPhones), as it's explicitely meant for powering 5V devices. 
```

Comment:
Would this include a charging circuit? I'd rather avoid having to take batteries out of the device for charging.